obs-packaging/linux-container/debian.rules

```diff
@@ -3,14 +3,23 @@ include /usr/share/dpkg/default.mk
 KernelDir=debian/kata-linux-container/usr/share/kata-containers
 DebugDir=debian/kata-linux-container-debug/usr/share/kata-containers
 KernelVer=${DEB_VERSION_UPSTREAM_REVISION}.container
+HOST_ARCH=$(shell arch)
+KATA_MULTIARCH="/usr/src/packages/SOURCES/kata-multiarch.sh"
```

Comment:
Q. is `kata-multiarch.sh` installed automatically?

Reply:
It is included in the packages sources at [obs-packaging/linux-container/kata-multiarch.sh](https://github.com/kata-containers/packaging/blob/e2fb50411f1ce54d9b54fa6f46b892f286f74b95/obs-packaging/linux-container/kata-multiarch.sh)

It is used only when bulding the kernel package, to determine those things that depends on architecture. Having this in a separate script make things cleaner in my view, and can be shared between spec file and debian rules.